tests/unit/binlog_test.cpp

```diff
@@ -0,0 +1,117 @@
+#define BOOST_TEST_DYN_LINK
+#define BOOST_TEST_MODULE DEFRAG library test
+
+#include <boost/test/included/unit_test.hpp>
+#include <boost/test/unit_test.hpp>
+
+#include "eblob_wrapper.h"
+
+#include "library/datasort.h"
+
+#include <vector>
```

Comment:
global includes should be placed before local

Reply:
fixed